Makefile

```diff
@@ -451,7 +469,13 @@ build/archives/$(NODEJS_PACKAGE_NAME):
 
 build/toolchain/nodejs/: build/archives/$(NODEJS_PACKAGE_NAME)
 	mkdir -p build/toolchain/nodejs/
+ifeq ($(suffix $(NODEJS_PACKAGE_NAME)),.zip)
+	# TODO: This is broken, there's the node-v10.15.3-win-x64 directory also windows does not have the bin/ directory.
```

Comment:
So for windows, what does 'broken' mean from the perspective of build experience?

Reply:
You can't build the rendered website. You can still use `make run-site` which is the Hugo preview mode. I verified that much.